tests/puppeteer/pages/BO/BO_login.js

```diff
@@ -0,0 +1,31 @@
+const Page = require('../page');
+
+module.exports = class BO_login extends Page {
+
+  constructor() {
+    super();
+
+    this.BO_login_pageTitle = 'Dashboard • PrestaShop';
```

Comment:
You're already in `BO_Login` context you can remove each occurrences of this

Reply:
I wanted to, but PHPStorm is *not* happy about this at all. It makes me do this : 
![image](https://user-images.githubusercontent.com/49909275/61276484-6dd0e480-a7b0-11e9-99c3-b864f839eeda.png)

Reply:
I mean:

```js
this.pageTitle = 'Dasho..';
this.emailInput = ...
```
And if you can be consistent and use camelcase or snake case only :)

Reply:
Oh right, the naming. In our test script, we will have a *lot* of page objects and selectors, so it seemed like a good idea to name all selectors with the page they're from. Even if they're prefixed with the page object's object.

Reply:
We changed the way we name selectors and used camelCase everywhere.